## Biological warfare between two bacterial viruses in an archipelago of defense systems sheds light on the spread of the CRISPR-Cas system

A plethora of bacterial defense mechanisms exist to combat infection by viruses and other external motile elements. A notable example is the CRISPR-Cas system, which is an adaptive immune system found in bacteria and archaea. Recent findings have revealed that these defense systems can be co-opted by viruses to enhance their ability to infect hosts successfully or to establish an adaptive synergy with the bacteria. In this study, we have conducted a comprehensive analysis of the viruses that infect Acinetobacter baumannii, an opportunistic pathogen that is a primary cause of nosocomial infections in hospital environments. Our research has identified one such virus, and further analysis revealed that this system is present in an integration hotspot of the bacterial genome, where it resides alongside numerous other defense mechanisms. The CRISPR-Cas system, being the I-Fa type mobilized by a virus known as PPTOP on some occasions, appears to be particularly targeted by another virus, called as DgiS1, that could compete with the system itself for the same integration site. It is noteworthy that both viruses possess a diverse array of defense mechanisms within its accessory genome, including Restriction-modification (RM) and RosmerTA systems, among others. Understanding of this viral warfare involving antiviral systems could be useful in the fight against infections caused by bacteria, and it would also shed light on how CRISPR-Cas systems expand in bacteria.

![imagen](https://github.com/user-attachments/assets/98d2f81f-1016-4995-b645-36b4a91baeaf)

1. Rubio A, Garzón A, Moreno-Rodríguez A, Pérez-Pulido AJ. Biological warfare between two bacterial viruses in a defense archipelago sheds light on the spread of CRISPR-Cas systems. Cell Rep. 2024 Dec 24;43(12):115085. doi: 10.1016/j.celrep.2024.115085. Epub 2024 Dec 13. PMID: 39675005.
2. Rubio A, Sprang M, Garzón A, Moreno-Rodriguez A, Pachón-Ibáñez ME, Pachón J, Andrade-Navarro MA, Pérez-Pulido AJ. Analysis of bacterial pangenomes reduces CRISPR dark matter and reveals strong association between membranome and CRISPR-Cas systems. Sci Adv. 2023 Mar 24;9(12):eadd8911. doi: 10.1126/sciadv.add8911. Epub 2023 Mar 24. PMID: 36961900; PMCID: PMC10038342.

### A. baumannii has two different subtypes of the CRISPR-Cas I-F system

The I-Fb system always appears integrated at exactly the same site, between the thiE gene (encoding a thiamine-phosphate synthase) and a gene encoding an uncharacterized protein (DUF962). However, the I-Fa system is found to be integrated with different frequencies at different sites

![imagen](https://github.com/user-attachments/assets/481e557c-76f8-456f-9a44-bd5333c390ee)

Fig. 1. Gene structure of the I-Fa and I-Fb CRISPR-Cas systems. Below the type of CRISPR-Cas system, the number of genomes that have it and the percentage with respect to the total number of genomes analyzed are shown. In the middle of both CRISPR-Cas systems, the percentage of identity between the homologous proteins is also shown and in parentheses is given the excess in number of amino acids that Cas proteins of the subtype I-Fb have compared with those of the I-Fa. The I-Fa subtype is usually integrated next to a tmRNA called ssrA, as a component of a genomic archipelago that may contain other genomic islands. The I-Fb subtype is always located between the thiE gene and an uncharacterized gene encoding a protein with a DUF962 domain.

In almost two-thirds of the genomes with I-Fa subtype, it is integrated near the 5′ end of the ssrA gene, a tmRNA. The I-Fa system can also appear in a circular molecule of about 115 kb, not assembled with the bacterial chromosome sequence. This fact suggests that it could actually be a phage-plasmid that occasionally carries a CRISPR-Cas I-Fa system. We named this phage-plasmid as Phage-Plasmid Terminator of Other Phages (PPTOP), based on the defense system it carries.

![imagen](https://github.com/user-attachments/assets/3a8aaff2-ed89-469c-bfd8-b9f1f5ac5bc0)

Fig. 2. Genomic regions in which CRISPR-Cas I-Fa systems are found. This subtype is mostly integrated next to the ssrA gene. However, it can also be found downstream of another gene cluster that includes a choline transporter, presenting only one of the cas genes (cas2_3), or integrated next to other genes to a lesser extent. Finally, it can also appear in a circular molecule. This configuration is also found in the p1DB053 plasmid (GenBank: CP087336.1) and in several phages (GenBank: BK017076.1; 30% coverage). Some of its plasmid genes are highlighted, as well as a phage integrase, several transposases on both ends of the CRISPR-Cas system, and a gene from a retron system. On the outer ring, the areas where the annotated phage and plasmid genes are located are marked, as well as the location of a direct repeat sequence (DR; ATGTAGAAATCTATATAGA).

### The ssrA gene provides support for an archipelago for the integration of multiple defense islands

The ssrA gene has been identified as a hotspot for the integration of genomic islands in bacteria. This is due to the integration of the CRISPR-Cas I-Fa system, as well as the presence of several genomic islands that have been integrated into it. The majority of these systems exhibited a high degree of diversity, suggesting that this genomic region may be regarded as a defense archipelago. Of note, the most recurrent genomic islands in this archipelago were two: the gene cluster containing the choline transporter and another with a putative prophage. This prophage, designated DgiS1, possesses a length of approximately 82 kilobases, exhibiting a guanine-cytosine content of 36.5% (whereas the average of A. baumannii is 38.5%), and has 90 genes.

![imagen](https://github.com/user-attachments/assets/b4026035-ed8d-4196-95cd-0bae0fa00696)

Fig. 3. Different genomic islands with the genes they contain. The colors highlight genes associated with specific defense systems. The numbers in parentheses accompanying genomic island names represent the absolute frequency of that genomic island next to the ssrA gene. The genomic islands are displayed in order of highest to lowest frequency. The most frequent variant of each island is shown.

### The prophage DgiS1 presents numerous defense systems against other prophages

Six general variants of the DgiS1 prophage have been identified, and these variants share a common genome that is related to essential phage genes. However, the accessory genome is predominantly comprised of an extensive battery of phage defense systems, encompassing Restriction-modification (RM) and RosmerTA systems.

![imagen](https://github.com/user-attachments/assets/88b425f0-0863-49b3-9c98-f8567782299f)

Fig. 4. Frequency of gene occurrence in DgiS1 clusters. The frequency of occurrence of the gene in each group or general variant of the DgiS1 prophage has been marked in different colors.

### The prophage DgiS1 and the CRISPR-Cas system do not usually co-exist in the same genome

Through an analysis of the islands integrated in the archipelago and the combination of these, it was observed that the DgiS1 prophage and the CRISPR-Cas I-Fa system do not usually appear together in the same genome, and when they do, the prophage appears mostly in an incomplete version (lack of genes).

![imagen](https://github.com/user-attachments/assets/e447318a-793f-47cc-8998-6c52d17a6094)

Fig. 5. Analysis of the islands integrated in the archipelago and the combination of these. (1) Frequency of the different combinations of genomic islands found. Only the most frequent combinations are shown. (2) Number of genomes having the DgiS1 prophage or the CRISPR-Cas I-Fa or I-Fb system, including group overlapping.

Comparing the genomes of the DgiS1-containing group with those containing the CRISPR-Cas I-Fa system, it was observed that the main difference involved the accessory genes of the opposite genomic island.

### The prophage DgiS1 could escape the CRISPR-Cas I-Fa system  by mutations in its PAM sequences

The presence of multiple spacers (I-Fa) against multiple sequence variants of the DgiS1 prophage suggests the potential for this virus to evade the CRISPR-Cas system. This outcome is facilitated by the prophage, which enables the fixation of mutations in the PAM sequence of the protospacer. While the primary PAM sequence remained CC, it exhibited diminished conservation, particularly at position -1, where a high rate of variation was observed.

![imagen](https://github.com/user-attachments/assets/7c5a3a24-79c1-42db-9b96-54931ae15526)

Fig. 6. DgiS1 prophage evasion of CRISPR I-Fa system. (1) The 10 nucleotides upstream of the protospacers of the prophage DgiS1 including the PAM sequence (positions −2 and −1). The number in parentheses shows the number of different protospacers found. (2) Frequencies of consensus PAM sequences in the protospacers of the prophage DgiS1. The y axis is in logarithmic scale.

